charts/kube-state-metrics/templates/servicemonitor.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
   {{- end }}
 spec:
   jobLabel: {{ default "app.kubernetes.io/name" .Values.prometheus.monitor.jobLabel }}
+  {{ include "servicemonitor.scrapeLimits" .Values.prometheus.monitor | nindent 2 }}
```

Comment:
but isn't this indenting the first line by 4 then and the rest by 2?

Reply:
Looks good on my side, do you have a different output?

```
$ helm template . | grep -C2 sampleLimit
  jobLabel: app.kubernetes.io/name
    
  sampleLimit: 1
  targetLimit: 2
  selector:
```

Reply:
but with indent it will ident the first line by 4 (the hardcoded 2 spaces and the extra spaces from the `indent`)
so then i have to revert it back to:
```
  jobLabel: {{ default "app.kubernetes.io/name" .Values.prometheus.monitor.jobLabel }}
{{ include "servicemonitor.scrapeLimits" .Values.prometheus.monitor | indent 2 }}
```

but i wanted to have the same solution like #2609

Reply:
Do you have 4 spaces on the first line when you build it?

Reply:
ah got it.. fixed :)